gradle/plugins/common/src/main/kotlin/junitbuild.java-library-conventions.gradle.kts

```
@@ -18,11 +18,15 @@ plugins {
 
 rewrite {
 	activeRecipe("org.openrewrite.java.migrate.UpgradeToJava17")
+	activeRecipe("org.openrewrite.staticanalysis.RemoveUnusedPrivateMethods")
+	activeRecipe("org.openrewrite.staticanalysis.MissingOverrideAnnotation")
```

assuming junit likes normalizations, the only question seems to be, is this limited to whitespaces, or does junit even care for some real issues being fixed?

Yes this particular recipe might be trivial, this is only to demonstrate and give some foundation to increment.

---

```suggestion
	activeRecipe("org.openrewrite.staticanalysis.MissingOverrideAnnotation")
	activeRecipe("org.openrewrite.java.testing.junit5.JUnit5BestPractices")
```
might want to ensure JUnit5BestPractices on JUnit, as well.